<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fereastra_principala</class>
 <widget class="QWidget" name="fereastra_principala">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>medii - disciplina</string>
  </property>
  <widget class="QPushButton" name="buton_d1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ECM 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton_d2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton_d3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PAC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton_d4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>SAIIV</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton_d5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>EG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton_d6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TSA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton_d7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IA 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buton_inapoi">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>550</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Inapoi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
